iphone/Classes/NetworkModule.h

```diff
@@ -23,7 +23,7 @@ typedef enum {
   TLS_VERSION_1_2
 } TLSVersion;
 
-@interface NetworkModule : TiModule {
+@interface NetworkModule : TiModule <UIApplicationDelegate> {
```

Comment:
Every thing looks good and working fine. One question -
If we have to use the UIApplicationDelegate methods in module, we have to include UIApplicationDelegate in module interface like below -

`@interface TestAppceleratorModule : TiModule<UIApplicationDelegate>
@end`

Isn't it better to include UIApplicationDelegate in TiModule ?

Reply:
The main reason was that I tried to not set the delegate if it's not actually used.